src/fluree/json_ld/processor/api.cljs

```diff
@@ -3,54 +3,76 @@
   (:require ["jsonld" :as jldjs]
             [fluree.json-ld.impl.external :as external]))
 
+(defn pluggable-loader
+  "Takes a document-loader, which takes a url and options and returns a json string
```

Comment:
Is this missing a conclusion or a comma? In other words, does "and returns a json string..." apply to the document-loader (in which case this is missing a conclusion about what the fn itself returns / does) or to the fn itself (in which case this is missing a comma between "and options" and "and returns" (or you could wrap "which takes a url and options" in parens).

Reply:
I see how that's confusing! I tried rewriting it a few times and then decided to just document the argument explicitly, I think it's an improvement.